dynacred/src/Contact.ts

```diff
@@ -52,6 +54,15 @@ export class Contact {
         return svBuf.readInt32LE(0);
     }
 
+    set structVersion(sv: number) {
```

Comment:
`Contact.structVersion = 2` gives `Contact.structVersion === 0`

Reply:
why?

Reply:
Oups, I read the second argument to `svBuf.writeInt32LE` as the value to save.